experimental/transliterate/tests/cldr.rs

```diff
@@ -12,11 +12,10 @@ include!("data/baked/mod.rs");
 
 #[test]
 fn test_all_cldr() {
-    let mut in_order =
-        std::fs::read_dir(concat!(env!("CARGO_MANIFEST_DIR"), "/tests/data/fixtures"))
-            .unwrap()
-            .map(|x| x.unwrap().path())
-            .collect::<Vec<_>>();
+    let mut in_order = std::fs::read_dir("tests/data/fixtures")
```

Comment:
Issue: This requires that the test is run in a certain directory, right? That's the whole point of using `CARGO_MANIFEST_DIR`.